src/ResourceManager/Resources/Commands.Resources/Models.Authorization/AuthorizationClient.cs

```diff
@@ -498,5 +500,66 @@ private static void ValidateRoleDefinition(PSRoleDefinition roleDefinition)
                 throw new ArgumentException(ProjectResources.InvalidActions);
             }
         }
+
+        public static void ValidateScope(string scope)
```

Comment:
yes.

---
In reply to: [120759096](https://github.com/Azure/azure-powershell/pull/4019#discussion_r120759096) [](ancestors = 120759096)